build-tests/api-documenter-test/etc/yaml/api-documenter-test/docclass1.yml

```diff
@@ -154,9 +155,57 @@ items:
       return:
         type:
           - 'api-documenter-test!SystemEvent:class'
+  - uid: 'api-documenter-test!DocClass1#seeExample:member(1)'
+    summary: Example of a `@see` tag.
+    seealsoContent: |-

```

Comment:
@rbuckton Does this DocFX field look correct to you? I don't have an easy way to test it.